frontend-react/src/layouts/Markdown/MarkdownLayout.tsx

```diff
@@ -0,0 +1,267 @@
+            {helmet}
+            <GridContainer className="usa-prose">
+                <Grid row className="flex-justify flex-align-start">
+                    {sidenavContent ? (
+                        <nav
+                            aria-label="side-navigation"
+                            className={`${styles.sidenav} tablet:grid-col-3`}
+                        >
+                            <MDXProvider
+                                {...mdx}
+                                components={{
+                                    ...MDXComponents,
+                                    ...mdx?.components,
+                                }}
+                            >
+                                {sidenavContent}
+                            </MDXProvider>
+                        </nav>
+                    ) : null}
+                    {main ?? (
+                        <main
+                            className={
+                                sidenavContent || tableOfContents
+                                    ? "tablet:grid-col-8"
+                                    : "tablet:grid-col-12"
+                            }
+                        >
+                            <header>
+                                {breadcrumbs != null ? (
+                                    <BreadcrumbBar>
+                                        {breadcrumbs.map((b) => (
+                                            <Breadcrumb key={b.label}>
+                                                {b.href ? (
+                                                    <USCrumbLink href={b.href}>
+                                                        {b.label}
+                                                    </USCrumbLink>
+                                                ) : (
+                                                    b.label
+                                                )}
+                                            </Breadcrumb>
+                                        ))}
+                                    </BreadcrumbBar>
+                                ) : null}
+                                <hgroup>
+                                    <h1>{title}</h1>
+                                    {sub}
+                                </hgroup>
+                                {callToAction
+                                    ? callToAction.map((c) => (
+                                          <CallToAction key={c.label} {...c} />
+                                      ))
+                                    : null}
+                                {lastUpdated ? (
+                                    <p className="text-base text-italic">
+                                        Last updated: {lastUpdated}
+                                    </p>
+                                ) : null}
+                            </header>
+                            {tableOfContents ? (
```

Comment:
can we use the `tableOfContents && (<Component />)` style instead?

Reply:
Agreed. Conditional renders simplified.